valora-inc/hooks
# feat(earn): Add logic to get TVL for dataProps

**Manual testing:**
Allbridge (see TVL [here](https://core.allbridge.io/pools?chain=CEL)):
```
yarn getPositions --networkId celo-mainnet --apps allbridge --address 0x644b896b0c45717215c0f10501b20fafd9bceb24
```
```
"dataProps": {
   "manageUrl": "https://core.allbridge.io/pools?chain=CEL",
   "contractCreatedAt": "2024-05-08T09:09:55.000Z",
   "tvl": 328777.712,
   ...
}
```
Aave (see total supplied [here](https://app.aave.com/reserve-overview/?underlyingAsset=0xaf88d065e77c8cc2239327c5edb3a432268e5831&marketName=proto_arbitrum_v3)):
```
yarn getPositions --networkId arbitrum-one --apps aave --address 0x644b896b0c45717215c0f10501b20fafd9bceb24
```
```
"dataProps": {
   "manageUrl": "https://app.aave.com/?marketName=proto_arbitrum_v3",
   "contractCreatedAt": "2023-06-28T10:09:48.000Z",
   "tvl": 204411266.810027,
    ...
}
```
